Cargo.toml

```diff
@@ -13,3 +13,5 @@ tokio = "0.3.5"
 tokio-amqp = "0.3.1"
 serde_json = "1.0.60"
 envy = "0.4.1"
+ed25519-dalek = "1.0.1"
+sha2 = "0.9.2"
```

Comment:
I can't seem to find the usage of this. Is it still needed or was it replaced by `ed25519-dalek`?

Reply:
yes I actually removed it locally but forgot to push 🤦

Reply:
resolved in 0b6ca95559e9bc25a01d42d4ca1a9a9bdeb0e266